### PR TITLE
Initial add Japanese language

### PR DESCRIPTION
**Pull request by mizunashi**
modify index.js
add ja.js

### CODE REVIEW

File: src/languages/index.js [hunk 1]
```diff
@@ -4,4 +4,9 @@ module.exports = [
         key: "en",
         table: require("./en"),
     },
+    {
+        name: "Japanese",
```
Comment (saleemrashid) [inline src/languages/index.js]: Can the language name be translated into Japanese? (i.e. `日本語`)

**Review by saleemrashid — CHANGES_REQUESTED** [review]
Please rebase this pull request (and make the change I mentioned).

Thank you very much!

File: src/languages/index.js (hunk 1 above)
Comment (mizunashi) [reply src/languages/index.js]: 日本語

**Comment by mizunashi** [comment]
Conflict resolution
I changed Japanese to 日本語.

File: src/index.html [hunk 2]
```diff
@@ -64,7 +64,7 @@ <h2>{{ "TITLE_DOWNLOAD_WALLET" | translate }}</h2>
                             <input type="password" ng-model="password" required>
                         </md-input-container>
                         <md-input-container class="md-block" flex>
-                            <md-button type="submit" ng-disabled="downloadForm.$invalid">{{ "BUTTON_DOWNLOAD_WALLET" | translate }}</md-button>
+                            <md-button type="submit" ng-disabled="downloadForm.$invalid">{{ BUTTON_DOWNLOAD_WALLET | translate }}</md-button>
```
Comment (saleemrashid) [inline src/index.html]: Could you revert this change?

**Review by saleemrashid — CHANGES_REQUESTED** [review]
Thanks! Just one more thing.

File: src/index.html (hunk 2 above)
Comment (mizunashi) [reply src/index.html]: I was afraid of revert's mistake, so I added "".
Is it not okay?

**Comment by saleemrashid** [comment]
@mizunashi Thanks for the awesome work! :+1:

**Comment by saleemrashid** [comment]
Rebuilt standalone HTML in a421076189cbd39cb82bdd1e0c0807dc69a24ec3
